### ISSUE TITLE
Error trying to rebuild model

### ISSUE THREAD

**Issue by palatos** [ThomasMerkh/covid-ct#2]
I'm trying to rebuild the model from the google docs files, but can't seem to.
I've tried a few different methods but I'm consistently getting the following error:

`Node 'gradients/post_bn/cond/FusedBatchNorm_grad/FusedBatchNormGrad' has an _output_shapes attribute inconsistent with the GraphDef for output #3: Dimension 0 in both shapes must be equal, but are 0 and 2048. Shapes are [0] and [2048].`

Do you know what might be causing this? Also I noticed there's a "model.data-00000-of-00001" but not a "model.data-00001-of-00001"

I'm more used to the Keras .h5 model save/load format, so I'm not sure if I'm doing something wrong here. Any advice is very welcome.

**Comment by hzhaoc** [comment]
Facing same issues. Let me know if this is solved

**Comment by balajimt** [comment]
This could be because you are using a different version of TensorFlow to train the model and a different version to evaluate. Try uninstalling the current TensorFlow version and then install `tensorflow 1.15.0` (as per requirements).
